src/components/page-elements/code.vue

```diff
@@ -19,32 +33,58 @@ export default {
   data() {
     return {
       copied: false,
+      supportsCB: false,
+      textButton: "Copy",
     };
   },
+  created() {
+    if (navigator.clipboard) {
+      this.supportsCB = true;
+    }
+  },
   computed: {
     coloredCode() {
       return this.code;
     },
   },
   methods: {
+    toggle() {
+      this.isActive = !this.enable;
+    },
     copyToClipboard,
     setCopied(copied) {
       this.copied = copied;
     },
+    async copyHandler() {
+      await copyToClipboard(this.coloredCode);
```

Comment:
Wrzuć to w blok try catch i wtedy wyświetlaj stosowną wiadomość czy się udało skopiować czy nie.